package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dataprotocol-client",
-  "version": "0.2.11",
+  "version": "0.2.15",
```

Comment:
this should be 0.2.12. Since we haven't done a 0.2.12 release it shouldn't cause any problems that you've used other numbers for local testing

Reply:
I have used the tags, i did 0.2.12 release and delete the release, but the tag has been used. after realizing that I have been using my own fork for testing.